### ISSUE TITLE
Are there any good forks of this project?

### ISSUE THREAD

**Issue by x13machine** [andredumas/techan.js#217]
Since the creator is dead, I hoping someone created an active fork. If there isn't I might do it, I dunno.

**Comment by rjarbour** [comment]
I added a fork for some random junk I was looking to add; Im not exactly talented, but I added a minor feature to plot precalculated data from the csv file with the techan.plot.arbitrary function. I also added some functionality to do moving averages over selected data rather than just the close more easily.

https://github.com/andredumas/techan.js/issues/205

If you want to work on an active fork please feel free to use mine.

**Comment by joaoapenas** [comment]
is the creator really dead?

**Comment by x13machine** [comment]
```
¯\_(ツ)_/¯
```

**Comment by counterbeing** [comment]
Issue #202 points to another library you might want to check out.

**Comment by joaoapenas** [comment]
i recently used c3 in my project, depending on what language you are using and what is the type of graph. I was working with timeseries chart and decided to use c3, the api is quite simple.

**Comment by counterbeing** [comment]
Yeah, I love c3, so simple, but it doesn't do candlestick charts, and it does seem to suffer with larger datasets.

**Comment by ColinEberhardt** [comment]
If people are looking for an alternative, d3fc (which I develop) has a very similar feature set. I've been expanding our gallery (https://d3fc.io/examples/) to show how it can be used to recreate a number of the techan examples:

<img width="706" alt="screen shot 2017-09-11 at 08 47 38" src="https://user-images.githubusercontent.com/1098110/30264387-6160a9ee-96d0-11e7-9939-7bcae1dc8244.png">
